CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+## [0.0.1] - 2020-07-01
+
+- Initial release
+### Added
```

Comment:
Instead of added, we should maybe just summarise what this initial release contains. This is basically what follows, but we can instead of Added maybe explain that we cut initial release to be able to track more carefully changes to the project with the following releases, and that at this point we don't have minimal set of features so it can be used in any reasonable scenario.